CONTRIBUTING.md

```diff
@@ -45,45 +45,75 @@ If you would like to contribute, please follow these steps:
+
+   Initialize the pre-commit hooks included within the root directory of `inundation-mapping`code folder:
     ```
     $ pre-commit install
     ```
-7. At this point, you should be set up with `pre-commit`. When a commit is made it will run the pre-commit hooks defined in [`.pre-commit-config.yaml`](.pre-commit-config.yaml). For reference, you may run any of the pre-commit hooks manually before issuing the `git commit` command (see below). Some tools used by the pre commit git hook (`isort`, `flake8`, & `black`) are also available as command line executables within the Docker container, however, it is recommend to run them through `pre-commit` outside of the container, as it picks up the correct configuration.
+    
+6. At this point, you should be set up with `pre-commit`. When a commit is made it will run the pre-commit hooks defined in [`.pre-commit-config.yaml`](.pre-commit-config.yaml). For reference, you may run any of the pre-commit hooks manually before issuing the `git commit` command (see below). Some tools used by the pre commit git hook (`isort`, `flake8`, & `black`) are also available as command line executables **within the Docker container***, however, it is recommend to run them through `pre-commit` outside of the container, as it picks up the correct configuration.
 
-    ```
-    # Check only the staged changes
-    pre-commit run
+   ```
+   # Check only the staged changes
+   pre-commit run
 
-    # Check all files in the repo
-    pre-commit run -a
+   # Check all files in the repo
+   pre-commit run -a
 
-    # Run only the flake8 formatting tool
-    pre-commit run -a flake8
-    ```
-
-8. Build the Docker container:
+   # Run only the flake8 formatting tool (or isort or black if you like)
+   pre-commit run -a flake8
+   ```
+   You can also run isort or black using the same pattern
+   ```
+   pre-commit run -a isort
+   pre-commit run -a black
+   ```
+7. Build the Docker container:
     ```
     Docker build -f Dockerfile -t <image_name>:<tag> <path/to/repository>
     ```
-9. [Within the container](README.md#startrun-the-docker-container), ensure sure unit tests pass ([instructions here](/unit_tests/README.md)).
+ 
+8.  [Within the container](README.md#startrun-the-docker-container), ensure sure unit tests pass ([instructions here](/unit_tests/README.md)).
     ```
     pytest unit_tests/
     ```
-
-10. Outside of the Docker container, commit your changes:
-    ```
-    git commit -m <descriptive sentence or two changes>
-    ```
-    This will invoke pre-commit hooks mentioned in step 7 that will lint & format the code. In many cases non-compliant code will be rectified automatically, but in some cases manual changes will be necessary. Make sure all of these checks pass, if not, make necessary changes and re-issue `git commit -m <...>`.
-
```

Comment:
Not sure it was intentional to remove this critical step.  Guidance is provided here on what the whole pre-commit system aims to accomplish.

Reply:
Nope. I must have lots in somewhere. I will put it back in.